ruby/webapp.rb.in

```diff
@@ -1,5 +1,20 @@
 plugin_path = "@CMAKE_SOURCE_DIR@/public" 
 
+#load dirs containing scripts
```

Comment:
Basically it is not a ruby package, its a html/js package.
What is getting installed is the ruby plugin, which is configured by cmake with the source location path.

But we have to als @jakobs about his original thoughts here.

Btw, installing the whole webapp would allow to install html pages from different plugins all into the ui folder, this whould render the plugin mechanism useless, we could install (and overwrite) files in the /ui folder which would make 
 https://github.com/rock-core/gui-rock_webapp/pull/2 unnessecary. On the other hand, I liked that a plugin can only add a new path to the server and NOT modify those of other plugins ot the original webapp.